metal/v1/model_vrf_route.go

```diff
@@ -454,17 +454,27 @@ func (o VrfRoute) MarshalJSON() ([]byte, error) {
 
 func (o VrfRoute) ToMap() (map[string]interface{}, error) {
 	toSerialize := map[string]interface{}{}
-	// skip: id is readOnly
-	// skip: status is readOnly
+	if !IsNil(o.Id) {
```

Comment:
The custom template was taken from OpenAPITools/openapi-generator#16525, so in addition to the unmarshal error change, it also includes a bug fix for serializing read-only fields.  We could instead create a custom template based on v6.6.0, but we want to upgrade to v7.0.0 anyway, so IMO we might as well keep this PR as-is.

Reply:
Alternatively we could get merged #148 first so that the IsNil change comes in as part of the upgrade and then rebase this on top of it.

Reply:
@ctreatma I suggest merging the v7.0.0 update first, if it shrinks the diff of this change (IIRC?).

Reply:
Generator change has been merged and the PR was rebased on `main`, so changes to generated code in this PR are specific to the template change under review.